### PR TITLE
Feat filetype icon inline files

### PR DESCRIPTION
**Pull request by Karim94**
#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/1036/mobile-inline-files-should-use-file-icons
#### Testing instructions  

----
### Repository owner

- [ ] Was tested and can be merged.
- [ ] PR name follows conventional changelog format.

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### CODE REVIEW

File: app/components/files/file-inline-container.js [hunk 1]
```diff
@@ -27,41 +32,52 @@ const descText = {
...
+            padding,
+            paddingBottom: (downloading && !isImage) ? paddingDifference : padding
+        };
         const header = {
             flexDirection: 'row',
             justifyContent: 'space-between',
-            marginBottom: isOpen ? 10 : 0
+            alignItems: 'center',
+            paddingBottom: isOpen ? (padding + borderWidth) : 0,
+            height: isOpen ? (containerHeight + padding) : containerHeight
```
Comment (seavan) [inline app/components/files/file-inline-container.js]: @Karim94 why do you need to set fixed height here? it usually should scale to content, that makes things much much easier to change.

Comment (Karim94) [reply app/components/files/file-inline-container.js]: Design specifies a 48 height for the container, what would be a better way to maintain that dimension, given this container is shared by inline files and images.

Comment (seavan) [reply app/components/files/file-inline-container.js]: alright. let's leave it for now as it is.

File: app/components/files/file-inline-container.js [hunk 2]
```diff
@@ -27,41 +32,52 @@ const descText = {
...
-                            { marginHorizontal: vars.spacing.small.maxi2x }
-                        )}
-                    </View>}
+            <View style={container}>
+                <View style={outer} {...this.props}>
+                    <View>
+                        {!!title && <Text style={titleText}>{title}</Text>}
+                        {!!description && <Text style={descText}>{description}</Text>}
+                    </View>
+                    <View style={[header, { marginBottom: !isImage && downloading ? 4 : 0 }]}>
```
Comment (seavan) [inline app/components/files/file-inline-container.js]: `4` should go to a constant.

**Comment by ebilg** [comment]
can be merged

**Pull request merged**